CMakeLists.txt

```diff
@@ -32,14 +32,14 @@ cmake_dependent_option(
 )
 
 find_path(HIDAPI_HEADER_DIR hidapi.h
```

Comment:
Should use pkg-config instead of raw find_path. If you do it without the Rock CMake macros, you could even propose the change upstream.